edgelet/edgelet-http/src/util/mod.rs

```diff
@@ -202,6 +203,7 @@ mod tests {
     }
 
     #[test]
+    #[cfg_attr(windows, ignore)] // remove when windows build servers are upgraded to RS5
```

Comment:
Prefix the comment with `TODO: ` please. Makes it easy to find them later.

Reply:
Will do